contracts/interfaces/IEverlongStrategy.sol

```diff
@@ -63,6 +72,22 @@ interface IEverlongStrategy is IPermissionedStrategy, IEverlongEvents {
     /// @return True if a new position can be opened, false otherwise.
     function canOpenPosition() external view returns (bool);
 
+    /// @notice Convert the amount of unwrapped tokens to the amount received
+    ///         after wrapping.
+    /// @param _unwrappedAmount Amount of unwrapped tokens.
+    /// @return _wrappedAmount Amount of wrapped tokens.
+    function convertToWrapped(
+        uint256 _unwrappedAmount
+    ) external view returns (uint256 _wrappedAmount);
+
+    /// @notice Convert the amount of wrapped tokens to the amount received
+    ///         after unwrapping.
+    /// @param _wrappedAmount Amount of wrapped tokens.
+    /// @return _unwrappedAmount Amount of unwrapped tokens.
+    function convertToUnwrapped(
+        uint256 _wrappedAmount
+    ) external view returns (uint256 _unwrappedAmount);
+
```

Comment:
We talked about this before, but are these any different from `convertToBase` or `convertToShares`? I'll return to this once I finish my review, but, a priori, I'm not sure I see why they'd be any different.

Reply:
They're not any different, the functions just add including a check for whether the strategy is using a wrapped token or not.

In terms of "understandability" I thought it'd be easier to follow using this language as opposed to `convertToBase` and `convertToShares`.

Reply:
Good call